webui.py

```diff
@@ -117,19 +128,24 @@ def removeModelCache():
     def updateExecutionProvider(provider):
         if provider == "cuda":
             if torch.cuda.is_available():
+                device = "cuda"
+                update_config(config, execution_provider=device)
```

Comment:
Remove duplicates, declare at the beginning or end

Reply:
Removed duplicates, declared them at the end